tests/distributions/test_categorical.py

```diff
@@ -80,19 +80,19 @@ def modify_params_using_dims(ps, dim):
 def test_support_dims(dim, ps):
     ps = modify_params_using_dims(ps, dim)
     support = dist.categorical.enumerate_support(ps)
-    assert_equal(support.size(), torch.Size((ps.size(-1),) + ps.size()[:-1] + (1,)))
+    assert_equal(support.size(), torch.Size((ps.size(-1),) + ps.size()[:-1]))
 
 
 def test_sample_dims(dim, ps):
     ps = modify_params_using_dims(ps, dim)
     sample = dist.categorical.sample(ps)
-    assert_equal(sample.size(), ps.size()[:-1] + (1,))
+    assert_equal(sample.size(), dist.categorical.shape(ps))
 
 
-def test_batch_log_dims(dim, ps):
-    batch_pdf_shape = (3,) + (1,) * dim
-    expected_log_pdf = np.array(wrap_nested(list(np.log(ps)), dim-1)).reshape(*batch_pdf_shape)
-    ps = modify_params_using_dims(ps, dim)
-    support = dist.categorical.enumerate_support(ps)
-    batch_log_pdf = dist.categorical.batch_log_pdf(support, ps)
-    assert_equal(batch_log_pdf.data.cpu().numpy(), expected_log_pdf)
+# def test_batch_log_dims(dim, ps):
```

Comment:
Remove or mark xfail.

Reply:
Yup, will do that. This is mostly to see if the integration tests pass.